app/Services/CustomerIo.php

```diff
@@ -184,7 +190,7 @@ public function deleteCustomer(string $id)
      */
     public function suppressCustomer(string $id)
     {
-        if (!config('features.delete-api')) {
+        if (!config('features.customer_io') && !config('features.delete-api')) {
```

Comment:
This should be an `||` so that if either of these are disabled, we skip running this code!

```suggestion
        if (!config('features.customer_io') || !config('features.delete-api')) {
```

Reply:
Ah, right. Then same for the `deleteCustomer()` above?

Reply:
Oh, yes! 👍